assets/src/edit-story/components/panels/utils/getBoundRect.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { dataPixels } from '../../../units/dimensions';
+
+/**
+ * Get the bound rect value for all objects in `list`,
+ *
+ * ```
+ *
+ * @param {Array.<Object>} list  List of objects
+ * @return {any} Returns common value or empty string if not similar
+ */
+function getBoundRect(list) {
+  const firstElementProperties = list[0].rotationAngle
+    ? calcRotatedObjectPositionAndSize(
+        list[0].rotationAngle,
+        list[0].x,
+        list[0].y,
+        list[0].width,
+        list[0].height
+      )
+    : list[0];
+  let startX = firstElementProperties.x;
+  let startY = firstElementProperties.y;
+  let endX = startX + firstElementProperties.width;
+  let endY = startY + firstElementProperties.height;
+
+  list.forEach((el) => {
+    const elementProperties = el.rotationAngle
+      ? calcRotatedObjectPositionAndSize(
+          el.rotationAngle,
+          el.x,
+          el.y,
+          el.width,
+          el.height
+        )
+      : el;
+    if (elementProperties.x < startX) startX = elementProperties.x;
```

Comment:
Could you pls redo this via `Math.min/max`?

Reply:
updated!